src/client/Client.js

```diff
@@ -414,6 +415,15 @@ class Client extends BaseClient {
       query.set('guild_id', guildID);
     }
 
+    if (options.additionalScopes) {
+      const scopes = options.additionalScopes;
+      if (scopes.some(scope => !InviteScopes.includes(scope))) {
+        throw new TypeError('INVALID_TYPE', 'additionalScopes', 'Array of Invite Scopes', true);
+      }
+      scopes.unshift('bot');
+      query.set('scope', scopes.join(' '));
```

Comment:
Not really a fan of using `Array#unshift()` here since it could end up mutating unintended arrays.
```suggestion
      query.set('scope', ['bot', ...scopes].join(' '));
```